mandatory/2-function-creation.js

```diff
@@ -25,7 +36,14 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  for (let i = 0; i < arrayOfStrings.length; i++) {
+    let trimmedString = arrayOfStrings[i].trim();
+    let lowerCased = trimmedString.toLowerCase();
+    arrayOfStrings[i] = lowerCased.split("/").join("");
```

Comment:
you can also use the [replace() method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace)